servicetalk-http-api/src/test/java/io/servicetalk/http/api/FormUrlEncodedHttpDeserializerTest.java

```diff
@@ -84,7 +84,42 @@ public void invalidContentTypeThrows() {
         final HttpHeaders headers = DefaultHttpHeadersFactory.INSTANCE.newHeaders();
         headers.set(CONTENT_TYPE, "invalid/content/type");
 
+        final String expectedMessage = "Unexpected headers, can not deserialize. Headers: DefaultHttpHeaders" +
```

Comment:
As we are only interested in making sure the desired content-type is present in the message, you can just check for the presence of that message:

```java
expectedException.expectMessage(containsString("invalid/content/type"));
```

This will limit changing tests if/when we change the exception message.

Reply:
This makes more sense :+1